scripts/admin/copy-dev-bundle-from-jenkins.sh

```diff
@@ -35,17 +35,17 @@ echo Found build $DIRNAME
 
 trap "echo Found surprising number of tarballs." EXIT
 # Check to make sure the proper number of each kind of file is there.
-s3cmd ls s3://com.meteor.jenkins/$DIRNAME/ | \
+aws s3 ls s3://com.meteor.jenkins/$DIRNAME/ | \
   perl -nle 'if (/\.tar\.gz/) { ++$TAR } else { die "something weird" }  END { exit !($TAR == 4) }'
 
 trap - EXIT
 
-for FILE in $(s3cmd ls s3://com.meteor.jenkins/$DIRNAME/ | perl -nlaF/ -e 'print $F[-1]'); do
-   if s3cmd info $TARGET$FILE >/dev/null 2>&1; then
+for FILE in $(aws s3 ls s3://com.meteor.jenkins/$DIRNAME/ | perl -nlaF/ -e 'print $F[-1]'); do
+   if [[ $(aws s3 ls $TARGET$FILE >/dev/null | wc -l) != 0 ]]; then
```

Comment:
*shrug* sure. 
It's widespread in the sense that I think aws cli is conistant across all it's calls, but return codes aren't really that standardized in unix. You'd expect it to act like "ls" which it doesn't quite.
A comment sounds reasonable, added.